src/helpers/stores/switchSpeedColors.js

```diff
@@ -1,9 +1,10 @@
+import config from '~helpers/configHelper';
 import { writable } from 'svelte/store';
 
 const LOCAL_STORAGE_KEY = 'switchSpeedColors';
 
 function getConfiguredValue() {
-  return window.localStorage.getItem(LOCAL_STORAGE_KEY) === 'true';
+  return JSON.parse(window.localStorage.getItem(LOCAL_STORAGE_KEY)) ?? config.SWITCH_COLORS;
```

Comment:
Why would `JSON.parse` "not be trustworthy"?
It has been implemented by every mature browser for decades (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#browser_compatibility) and parses JSON to objects, its how JSON works...

Reply:
Not be thrustworthy might not be the best formulation in this case. I'm mostly looking for more readable code. The fact that `JSON.parse('true')` results in `true` and `JSON.parse('false')` results in `false` is quite abstract when trying to interpet this code in my head.

Personally I'm a big believer of making code readable rather than smart. But it's definitely opionated, we can leave it as it is and if needed it can be adapted in the future.